### ISSUE TITLE
Error UUID of null in Accessory.addService

### ISSUE THREAD

**Issue by chrissearle** [ptz0n/homebridge-verisure#10]
Just testing out the plugin.

It initializes 3 smoke detectors, the main controller, the speaker but then it finds a PIR2:

```
[4/8/2017, 11:23:57 AM] [Verisure] Device PIR2 is not yet supported
/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/Accessory.js:128
    if (existing.UUID === service.UUID) {
                                 ^

TypeError: Cannot read property 'UUID' of null
    at Accessory.addService (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/Accessory.js:128:34)
    at /usr/local/lib/node_modules/homebridge/lib/server.js:418:19
    at Array.forEach (native)
    at Server._createAccessory (/usr/local/lib/node_modules/homebridge/lib/server.js:402:14)
    at Server.<anonymous> (/usr/local/lib/node_modules/homebridge/lib/server.js:364:32)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/util/once.js:16:19
    at /usr/local/lib/node_modules/homebridge-verisure/index.js:111:9
    at Request._callback (/usr/local/lib/node_modules/homebridge-verisure/node_modules/verisure/index.js:59:7)
    at Request.self.callback (/usr/local/lib/node_modules/homebridge-verisure/node_modules/request/request.js:188:22)
    at emitTwo (events.js:106:13)

```

This makes homebridge abort startup.

**Comment by ptz0n** [comment]
Hi there, what kind of device is a `PIR2`?

**Comment by chrissearle** [comment]
I assume its the PIR with inbuilt camera - see https://www.verisure.no/images/18.361022c3158b9dba0ad7362d/1485335313115/fotodetektor_500x500.png

**Comment by ptz0n** [comment]
Oh, I see. What kind of data can you get from it?

**Comment by chrissearle** [comment]
Based on what I can see in the verisure gui - not a lot. Not even the temperature reading I see from the older style non-camera PIR. So I think what we likely need to do here is ignore the sensor if seen - instead of throw an exception.

**Comment by ptz0n** [comment]
Agree, that would be a better approach.

**Comment by ptz0n** [comment]
We are now ignoring any non-supported devices. Closing.

**Comment by chrissearle** [comment]
@ptz0n Sorry that I never managed to test this - my home install broke and it's only just come back.

I've tried with master - and it still fails.

I still do get the Device PIR2 is not yet supported.

Looking at the lastest copy of index.js - https://github.com/ptz0n/homebridge-verisure/blob/master/index.js#L348-L352 - I can see that even if we have this - we still return service in line 352.

So - if I understand it correctly - we will still hit this error if PIR2 is added to the list of accessories that gets sent to the callback in https://github.com/ptz0n/homebridge-verisure/blob/master/index.js#L88-L130 ?

From what I can see - homebridge will call the VerisurePlatform.accessories method with a callback and we send in _all_ detected accessories.

Each accessory has a getServices() method - and in here we do things only for the ones we know - but by this point we've already added the accessory to the list homebridge is expecting?

Can't see how the "non-supported" devices are getting ignored - but I haven't worked with either verisure or homebridge APIs before - might be missing it.
